pkg/udf/reduce_udf.go

```diff
@@ -61,29 +61,29 @@ func (u *ReduceUDFProcessor) Start(ctx context.Context) error {
 	var reader isb.BufferReader
 	var writers map[string]isb.BufferWriter
 	var err error
+	var fromBuffer dfv1.Buffer
 	fromBuffers := u.VertexInstance.Vertex.GetFromBuffers()
-	var fromBufferName string
 	for _, b := range fromBuffers {
```

Comment:
```suggestion
	// choose the buffer that corresponds to this reduce processor because
	// reducer's incoming edge can have more than one buffer for parallelism 
	for _, b := range fromBuffers {
```